external/common_functions.sh

```diff
@@ -219,6 +226,23 @@ function set_test_info() {
         ubi_packages="git wget unzip"
         ubi_minimal_packages="${ubi_packages}"
         ;;
+    external_custom)
+        PROPERTY_FILE=external_custom/test.properties
+        GITHUB_URL=$(getProperty "github_url")
+        github_url="${EXTERNAL_CUSTOM_TARGET}"
+        script="test.sh"
+        test_results="testResults"
+        tag_version="master"
```

Comment:
tag_version="${EXT_REPO_BRANCH}"

Reply:
I have made the required changes for the EXT_REPO_BRANCH and successfully run the spring tests too which couldn't be run earlier due to branch name issues.
![spring](https://user-images.githubusercontent.com/59281636/123113206-fd32a800-d45b-11eb-9af6-c2d5a49237ea.png)